cea/technologies/solar_collector.py

```diff
@@ -657,11 +657,11 @@ def Calc_categoriesroof(teta_z, B, GB, Max_Isol):
 
 # investment and maintenance costs
 
-def calc_Cinv_SC(Area, gv):
+def calc_Cinv_SC(Area_m2, gv):
     """
     Lifetime 35 years
     """
-    InvCa = 2050 * Area / gv.SC_n  # [CHF/y]
+    InvCa = 2050 * Area_m2 / gv.SC_n  # [CHF/y]
```

Comment:
actually, the other PR about the investment cost changes this format and represents the cost in two terms `Capex_a` and `Opex_fixed`. So i think we can make the changes in the other PR to avoid repetition